web/containers/admin/AssessmentPage/index.js

```diff
@@ -9,6 +9,8 @@ import { convertObjectPrimitivesToStrings } from '../../../util/object';
 import AssessmentPage from '../../../components/admin/AssessmentPage';
 import applicantCollections from '../../../util/applicantCollections';
 
+const Json2csvParser = require('json2csv').Parser;
```

Comment:
You can make this an ES6 import like the rest of the imports by doing something like `import { Parser } from 'json2csv'` and replacing `new Json2csvParser` on L228 with `new Parser`.

Reply:
great, I updated it